cfme/infrastructure/deployment_roles.py

```diff
@@ -71,59 +253,57 @@ def step(self):
         except NoSuchElementException:
             nav_select('Clusters / Deployment Roles')
 
-    def am_i_here(self):
-        def match(option):
-            return match_page(title=option, summary=option)
-
-        return match('Clusters / Deployment Roles') or match('Deployment Roles')
+    def resetter(self):
```

Comment:
**REQUIRED**
You don't need the resetter and all stuff below when you are using entities.